guides/doc-Administering_Red_Hat_Satellite/topics/Maintenance.adoc

```diff
@@ -74,6 +74,11 @@ For example, to schedule the cron job to run every day at 15:00, enter the follo
 [[sect-Administering-Recovering_from_a_Full_Disk]]
 === Recovering from a Full Disk
 
+ifndef::satellite[]
```

Comment:
Can't we hide this on non-Katello builds? I think it's terrible UX to have to skip over content and we have the ability to generate multiple documents.

Also, isn't there a Katello tag? That's probably nicer for Orcharhino as well.

Reply:
We discussed this in the past and we thought that a note is better than hiding whole chapters because it presents the users whats possible if they had installed Katello. At least this is what we were doing everywhere, if we want to change the strategy then we should probably do it everywhere.

I kinda lean towards showing it and advocating Katello deployments.